awesome.c

```diff
@@ -131,6 +131,8 @@ awesome_atexit(bool restart)
     /* Close Lua */
     lua_close(L);
 
+    screen_cleanup();
```

Comment:
Because that's actually the "last" real leak in during the tests if fontconfig and dbus are added to a suppression file. The leak checks come with "Address sanitizer" unless you disable it. I plan to enable ASAN in the CI eventually (it catches a lot more than just leaks). Then fixing the leaks makes sense. And, yes, I am aware there's a way to disable the memory leak component from ASAN, but then you also lose real leaks.